kitchen.yml

```diff
@@ -16,17 +17,54 @@ platforms:
       centos-7.1
       debian-7.8
     )
+    # ERROR: Cookbook 'apt' version '7.3.0' depends on chef version [">= 13.3"], but the running chef version is 12.7.2
     chef_versions = %w(
       12.7.2
       13.11.3
       14
       15
     )
+    windows_platform = 'windows-2012r2'
+    windows_chef_versions = %w(
+      14.14.29
+    )
+    platforms_combo = {}
+%>
 
-    test_platforms.product(chef_versions).each do |platform_version, chef_version|
+.WINDOWS_PLATFORMS: &WINDOWS_PLATFORMS
+<% windows_chef_versions.each do |chef_version| %>
+  - <%= windows_platform %>-<%= chef_version %>
+<% end %>
+
+.LINUX_PLATFORMS: &LINUX_PLATFORMS
+<% test_platforms.product(chef_versions).each do |platform_version, chef_version| 
+  platform_name = platform_version[/(.*)-(.*)/, 1]
+  platform_version = platform_version[/(.*)-(.*)/, 2]
+  uri = URI.parse("https://omnitruck.chef.io/stable/chef/metadata?v=#{chef_version}&p=#{platform_name}&pv=#{platform_version}&m=x86_64")
+  puts "Checking #{uri}"
+  resp = Net::HTTP.get_response(uri)
+  puts resp.code == "200"
```

Comment:
This is quite spammy (two lines per os*chef) and, the first time I ran it, quite slow (subsequent runs were faster for some reason but still not instant) which is quite annoying when you are just running a `kitchen list` or another command that should be fast.

Since the output will be always the same, can we hardcode the list of supported os*chef products instead of calculating it every time? Or calculate it once and cache it, if you prefer (but it's more work, not sure it's worth it).